purchase_vendor_bill_breakdown/readme/USAGE.rst

```diff
@@ -0,0 +1,9 @@
+* Enable **Use Product Components** option in the **Contact** section of the  **Sales & Purchases** tab.
+* Enable **Use Product Components** option in the **Products** section of the **Purchase** tab.
+* In the **Purchase** tab of the **Product**  there will be a button added for the seller.
+* Click the button and add components for the product.
+* Create a new RQF from this Vendor
+* Hint: you can disable/enable components in the **Other information** tab of the RFQ
+* Confirm the RFQ. Not there will be a button for the component wizard in the PO Lines
```

Comment:
* Confirm the RFQ. Note there will be a button for the product component breakdown in the PO Lines, tracking qty received and billed